### DISCUSSION TITLE
Fabric & Gemini 1.5 Pro

### DISCUSSION THREAD

**Discussion by GhiathTheDev** [danielmiessler/Fabric#637] [Q&A] [answered]
Hi, 
  I have install Fabric in my Debian machine and provided the Gemini-API key during the setup. When i try to execute a Fabric pattern, I'm getting the following error message:
  `Error: Error code: 404 - {'error': {'message': 'The model `gemini-1.5-pro:latest` does not exist or you do not have access to it.', 'type': 'invalid_request_error', 'param': None, 'code': 'model_not_found'}}`

Even though the --listmodel command shows the following models available:
GPT Models:
gpt-3.5-turbo
gpt-3.5-turbo-0125
gpt-3.5-turbo-1106
gpt-3.5-turbo-16k
gpt-3.5-turbo-instruct
gpt-3.5-turbo-instruct-0914

Local Models:

Claude Models:

Google Models:
models/gemini-1.0-pro
models/gemini-1.0-pro-001
models/gemini-1.0-pro-latest
models/gemini-1.0-pro-vision-latest
models/gemini-1.5-flash
models/gemini-1.5-flash-001
models/gemini-1.5-flash-latest
models/gemini-1.5-pro
models/gemini-1.5-pro-001
models/gemini-1.5-pro-latest
models/gemini-pro
models/gemini-pro-vision

What I'm missing or doing wrong?

Thank for all your help in advance :)

**Comment by snafu4** [comment, marked as answer]
Try adding ‘models/‘ to the front of your model name to match listmodels output.

**Reply by GhiathTheDev** [reply]
Thanks. That was it.
